src/Summary/SummaryView.js

```diff
@@ -3,6 +3,7 @@ import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
```

Comment:
Please, import only functions that you need to use. In your case ‘import { get } from ‘lodash’